README.md

```diff
@@ -1,30 +1,40 @@
 <div align="center">
-<img  align="center" src="./docs/logo.png" alt="Exposed" width="315" /></div>
+
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/logo-dark.png" width="315">
+    <img alt="Exposed logo" src="./docs/logo-light.png" width="315">
+  </picture>
+
+</div>
```

Comment:
Just an option to make the front page cleaner in dark mode. It does mean introducing a second .png (both are around 51kb, while the original was 41kb). Maybe it would be better to wait for svgs.

Please let me know if we should wait. I got the screenshot I needed from the preview anyway 😄